### ISSUE TITLE
Подключить VK

### ISSUE THREAD

**Issue by alexzaporozhets** [libertyads/marketplace#38]
по аналогии с facebook

**Comment by orestx** [comment]
Подключить учетную запись получилось.
Но вот насчет публикации нюанс. Она работает. но что бы опубликовалось нужно дополнительное действие - нажать "опубликовать". Я в курсе что без этого никак. Но нужно как то дать знать об этом пользователю.
Сообщения "You offer was paid"  в этом случае не достаточно. Нужно еще дописать ... please click "publish"  to complete your deal.
Текст можно еще обсудить, но что то такое необходимо.

**Comment by orestx** [comment]
И вообще, можно при подключении канала ВК выводить модальное окно в котором будет текст "Вы успешно подключили свой канал ВК, но к сожалению мы не можем автоматически публиковать сообщений от ваших сделок, поэтому после получения оплаты, для получения вознаграждения , вам нужно будет каждый раз нажимать кнопку `опубликовать`". Или типа того, что бы пользователь сразу знал что к чему и не расслаблялся после того как дал согласие на публикацию.

**Comment by maximsheiko** [comment]
да, я тоже об этом думал, либо сделать это в виде подсказки где то, в общем как скажете так и сделаю

**Comment by orestx** [comment]
@maximsheiko 
Первым делом пофиксь баг с оплатой биткоином.

**Comment by maximsheiko** [comment]
так нормально?
![image](https://cloud.githubusercontent.com/assets/10918691/8774840/4825e1fa-2eeb-11e5-8dfa-8f40327f43db.png)

**Comment by orestx** [comment]
После button поставь точку. Так же мне кажется что тут:
>...so after receiving the payment for an award,...

нужно еще одну запятую, после `payment`, по крайней мере у меня получается не тот смысл и я бы лучше написал `to receive a reward` вместо `for an award`. Или `to get reward`. В общем нужна помощь зала. Сейчас еще уточним.

**Comment by maximsheiko** [comment]
http://picscomment.com/pics/50441430837115.jpg
